include/slang/ast/symbols/ClassSymbols.h

```diff
@@ -71,6 +71,18 @@ class SLANG_EXPORT ClassType : public Type, public Scope {
         return implementsIfaces;
     }
 
+    /// Gets the list of interface classes that this class implements.
+    /// If this class is itself an interface class, this is instead the list of
+    /// interface classes that it extends from, if any.
+    ///
+    /// Note that this list is *not* flattened from the full set of all interfaces implemented
+    /// by any base classes or interface class parents, up the inheritance hierarchy.
+    /// It only includes those mentioned as being implemented in the class definition.
+    span<const Type* const> getImplementedInterfacesImmediate() const {
```

Comment:
I would call this "getDirectlyImplementedInterfaces" instead of using the "immediate" term.